src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.VisualBasic.props

```diff
@@ -13,6 +13,14 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <PropertyGroup>
     <VBRuntime Condition=" '$(VBRuntime)' == '' ">Embed</VBRuntime>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <WarningLevel Condition=" '$(WarningLevel)' == '' ">1</WarningLevel>
+    <NoWarn Condition=" '$(NoWarn)' == '' ">41999,42016,42017,42018,42019,42020,42021,42022,42032,42036</NoWarn>
+    <OptionExplicit Condition=" '$(OptionExplicit)' == '' ">On</OptionExplicit>
+    <OptionCompare Condition=" '$(OptionCompare)' == '' ">Binary</OptionCompare>
+    <OptionStrict Condition=" '$(OptionStrict)' == '' ">Off</OptionStrict>
+    <OptionInfer Condition=" '$(OptionInfer)' == '' ">On</OptionInfer>
+    <DisableImplicitNamespaceImports Condition="'$(DisableImplicitNamespaceImports)'==''">$(DisableImplicitFrameworkReferences)</DisableImplicitNamespaceImports>
```

Comment:
This line (setting `DisableImplicitNamespaceImports`) should probably go in the .targets instead of the .props.  People will set `DisableImplicitFrameworkReferences` in the body of their project, but if this line is here then that won't affect the default value of `DisableImplicitNamespaceImports`.